spring/boot-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaSslConfigurationTest.java

```diff
@@ -0,0 +1,101 @@
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.DisableOnDebug;
+import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.google.common.collect.ImmutableList;
+
+import com.linecorp.armeria.client.HttpClient;
+import com.linecorp.armeria.common.AggregatedHttpMessage;
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.server.AbstractHttpService;
+import com.linecorp.armeria.server.PathMapping;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServiceRequestContext;
+import com.linecorp.armeria.server.logging.LoggingService;
+import com.linecorp.armeria.spring.ArmeriaAutoConfigurationTest.TestConfiguration;
+
+/**
+ * This uses {@link ArmeriaAutoConfiguration} for integration tests.
+ * application-sslTest.yml will be loaded with minimal settings to make it work.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = TestConfiguration.class)
+@ActiveProfiles({ "local", "sslTest" })
+@DirtiesContext
+public class ArmeriaSslConfigurationTest {
+
+    @SpringBootApplication
+    public static class TestConfiguration {
+
+        @Bean
+        public HttpServiceRegistrationBean okService() {
+            return new HttpServiceRegistrationBean()
+                    .setServiceName("okService")
+                    .setService(new OkService())
+                    .setPathMapping(PathMapping.ofExact("/ok"))
+                    .setDecorators(ImmutableList.of(LoggingService.newDecorator()));
+        }
+    }
+
+    public static class OkService extends AbstractHttpService {
+        @Override
+        protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req) throws Exception {
+            return HttpResponse.of(HttpStatus.OK, MediaType.PLAIN_TEXT_UTF_8, "ok");
+        }
+    }
+
+    @Rule
+    public TestRule globalTimeout = new DisableOnDebug(new Timeout(10, TimeUnit.SECONDS));
+
+    @Inject
+    private Server server;
+
+    private String newUrl(String scheme) {
+        final int port = server.activePort().get().localAddress().getPort();
```

Comment:
Because the port is dependent on the order which is specified in the property file, how about bringing it explicitly as we do in [`ServerRule`](https://github.com/line/armeria/blob/armeria-0.82.0/testing/src/main/java/com/linecorp/armeria/testing/server/ServerRule.java#L167)?